kthresher.py

```diff
@@ -213,6 +213,8 @@ def kthreshing(purge=None, headers=None, keep=1):
                ):
                 if pkg.installed.version in kernels.keys():
                     kernels[pkg.installed.version].append(pkg.name)
+                else:
+                    kernels[pkg.installed.version] = [pkg.name]
```

Comment:
Might be safer to append instead of assign a new list, maybe not needed at all, but thinking about the future.